gcloud-java-contrib/gcloud-java-nio-examples/src/main/java/com/google/gcloud/nio/examples/ListFilesystems.java

```diff
@@ -43,4 +43,5 @@ private static void listFilesystems() {
       System.out.println("  " + p.getScheme());
     }
   }
+
 }
```

Comment:
OK, I put it back. I believe that some editors like to strip (or add) a final \n. I can't imagine it would ever matter, though.